integration/tests/utils/util-functions.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021-2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Are we using every function defined in this file? If not, let's remove them, no?